### DISCUSSION TITLE
traefik image not processed

### DISCUSSION THREAD

**Discussion by vrbgtrhr** [rancher/rke2#7288] [Q&A]
i have placed rke2-images.linux-amd64.tar.zst and rke2-images-traefik.linux-arm64.tar.zst in /var/lib/rancher/rke2/agent/images/ but the traefik pod that spin up using the following config.yaml give ErrImagePull
```
ingress-controller: traefik
disable:
  - rke2-ingress-nginx
```

Version: v1.31.2+rke2r1

**Comment by dereknola** [comment]
>  rke2-images.linux-amd64.tar.zst and rke2-images-traefik.linux-arm64.tar.zst

You are using two different architectures for your images. Try it again with both `amd64`

**Comment by vrbgtrhr** [comment]
it still try to pull the image from the registry even if during the install.sh i get:
```
[INFO]  staging local checksums from /home/rke2/sha256sum-amd64.txt
[INFO]  staging zst airgap image tarball from /home/rke2/rke2-images.linux-amd64.tar.zst
[INFO]  staging tarball from /home/rke2/rke2.linux-amd64.tar.gz
[INFO]  verifying airgap tarball
[INFO]  installing airgap tarball to /var/lib/rancher/rke2/agent/images
[INFO]  Installing airgap image from /home/rke2/rke2-images-traefik.linux-amd64.tar.zst
[INFO]  verifying tarball
[INFO]  unpacking tarball file to /usr/local
```
and `ls /var/lib/rancher/rke2/agent/images/` return:
rke2-images.linux-amd64.tar.zst  rke2-images-traefik.linux-amd64.tar.zst

**Comment by brandond** [comment]
The image tarballs need to be in `/var/lib/rancher/rke2/agent/images` when RKE2 starts. The install script doesn't do anything with them, they are processed by RKE2 when it starts. If you put the tarballs there **after** starting RKE2, they will not be imported.

Check to see what images you have imported:
`CRI_CONFIG_FILE=/var/lib/rancher/rke2/agent/etc/crictl.yaml /var/lib/rancher/rke2/bin/crictl image ls`

**Comment by vrbgtrhr** [comment]
ok i think that the problem is that the pod try to get version 2.10.7 but is present 2.11.10

**Comment by vrbgtrhr** [comment]
i have tried to fix the problem by doing so:

```
docker pull rancher/mirrored-library-traefik:2.10.7
docker save rancher/mirrored-library-traefik -o rke2-images-traefik.linux-amd64
tar -I zstd -cvf rke2-images-traefik.linux-amd64.tar.zst rke2-images-traefik.linux-amd64
```

then i used the new file removing the old one but now the image is no where to be found
